src/mame/drivers/viper.cpp

```diff
@@ -98,7 +98,7 @@
         p911(all)           Goes in-game. Controllers are not emulated. Various graphical glitches.
         tsurugi,j           Goes in-game. Controllers are not emulated. Various graphical glitches.
 
-        gticlub2,ea         Attract mode works. Coins up. Hangs in various places. Will crash with "network error" after stage is selected.
+        gticlub2,drivparty  Attract mode works. Coins up. Hangs in various places. Will crash with "network error" after stage is selected.
```

Comment:
The part after the comma in this list is clearly the region that the comment applies to.  Don’t break it.

Reply:
The comment applies to both `gticlub2` and `gticlub2ea`. Whoever wrote the initial comment just did it in a shorthand lazy way that is meant to be `gticlub2 and gticlub2ea`.